### PR TITLE
LOOP-28 Upload Loop settings data to Tidepool backend

### PR DESCRIPTION
**Pull request by darinkrauss**
- https://tidepool.atlassian.net/browse/LOOP-28
- Improve StoredSettings storage
- Make public actionDuration and peakActivity of ExponentialInsulinModelPreset

### CODE REVIEW

**Review by nhamming — APPROVED** [review]
LGTM

File: Loop/Managers/LoopDataManager.swift [hunk 1]
```diff
@@ -684,26 +684,24 @@ extension LoopDataManager {
...
-            settings.glucoseUnit = loopSettings.glucoseUnit
-            settings.deviceToken = loopSettings.deviceToken
-            settings.bundleIdentifier = Bundle.main.bundleIdentifier
-
-            settings.insulinModel = UserDefaults.appGroup?.insulinModelSettings?.model
-            settings.basalRateSchedule = UserDefaults.appGroup?.basalRateSchedule
-            settings.insulinSensitivitySchedule = UserDefaults.appGroup?.insulinSensitivitySchedule
-            settings.carbRatioSchedule = UserDefaults.appGroup?.carbRatioSchedule
-
+        if let appGroup = UserDefaults.appGroup, let loopSettings = appGroup.loopSettings {
```
Comment (rickpasetto) [inline Loop/Managers/LoopDataManager.swift]: [nit] consider 
```suggestion
        guard let appGroup = UserDefaults.appGroup, let loopSettings = appGroup.loopSettings else { return }
```

**Review by rickpasetto — APPROVED** [review]
One very small nit, other than that LGTM 🏄

**Review by rickpasetto — APPROVED** [review]
LGTM 🌵

**Pull request merged**